exercises/complex-number/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct Complex {
+    // Add in the fields
+}
+
+impl Complex {
+    pub fn new(_re: f64, _im: f64) -> Complex {
+        unimplemented!()
+    }
+
+    pub fn exp(&self) -> Complex {
+        unimplemented!()
+    }
+
+    // Return the real part
+    pub fn re(&self) -> f64 {
+        unimplemented!()
+    }
+
+    // Return the imaginary part
+    pub fn im(&self) -> f64 {
+        unimplemented!()
+    }
+
+    // Return the absolute value
+    pub fn abs(&self) -> f64 {
+        unimplemented!()
+    }
+    
+    // Return the conjugate
+    pub fn conjugate(&self) -> Complex {
+        unimplemented!()
+    }
+}
+
+// Implement `std::ops` for `Complex`
```

Comment:
I want to invite discussion from the other maintainers: should we provide stubs for the ops functions, or require students to look up and import them?

In the [decimal exercise](https://github.com/exercism/rust/tree/master/exercises/decimal) we do not provide the stubs, but that is significantly later in the track. I could be persuaded in either direction for this exercise. What do you guys think?

Note that because that exercise doesn't include the stubs, it requires the use of `.meta/ALLOWED_TO_NOT_COMPILE`; this would also require that in case we choose to take out these stubs.